src/SFA.DAS.Reservations.Web/Models/ManageViewModel.cs

```diff
@@ -10,5 +10,7 @@ public class ManageViewModel
         public ManageReservationsFilterModel FilterModel { get; set; }
         public bool ShowSearch => TotalReservationCount >= ReservationsWebConfigurationConstants.NumberOfReservationsRequiredForSearch;
         public int TotalReservationCount { get; set; }
+        public bool ShowFilteredSearch =>

```

Comment:
Can this be renamed to `ShowPageLinks` please?

Reply:
Done